# Copyright (c) 2013-2018 Hanson Robotics, Ltd, all rights reserved 
expressions:
  type: array
  format: table
  items:
    type: object
    properties:
      name:
        title: 'Name'
        propertyOrder: 1
        type: string
      probability:
        title: 'Probability'
        propertyOrder: 2
        type: number
        default: 0.1
      magnitude_min:
        title: 'Min. Magnitude'
        propertyOrder: 3
        type: number
        default: 0.5
      magnitude_max:
        title: 'Max. Magnitude'
        propertyOrder: 4
        type: number
        default: 1.0
      duration_min:
        title: 'Min. Duration'
        propertyOrder: 5
        type: number
        default: 1.0
      duration_max:
        title: 'Max. Duration'
        propertyOrder: 6
        type: number
        default: 4.0
gestures:
  type: array
  format: table
  items:
    type: object
    properties:
      name:
        title: 'Name'
        propertyOrder: 1
        type: string
      probability:
        title: 'Probability'
        propertyOrder: 2
        type: number
        default: 0.1
      magnitude_min:
        title: 'Min. Magnitude'
        propertyOrder: 3
        type: number
        default: 0.5
      magnitude_max:
        title: 'Max. Magnitude'
        propertyOrder: 4
        type: number
        default: 1
      speed_min:
        title: 'Min. Speed'
        propertyOrder: 5
        type: number
        default: 0.6
      speed_max:
        title: 'Max. Speed'
        propertyOrder: 6
        type: number
        default: 1
